src/pages/index.js

```diff
@@ -26,29 +28,80 @@ const profilePopup = new PopupWithForm(
 profilePopup.setEventListeners();
 const newCardPopup = new PopupWithForm("#newcards", addNewCardSubmit);
 newCardPopup.setEventListeners();
+const EditPfp = new PopupWithForm("#pfpEdit", editPfpSubmit);
```

Comment:
`EditPfp`


Every variable in the project should be in `camelCase` (in **lower case** if it's not a class or component)


And everyone should understand what it is


`avatarPopup`